api/desecapi/serializers.py

```diff
@@ -1,6 +1,71 @@
 from rest_framework import serializers
-from desecapi.models import Domain, Donation, User
+from desecapi.models import Domain, Donation, User, RRset
 from djoser import serializers as djoserSerializers
+import json
+
+
+class JSONSerializer(serializers.Field):
+    def to_representation(self, obj):
+        return json.loads(obj)
+
+    def to_internal_value(self, data):
+        return json.dumps(data)
+
+
+class RecordsSerializer(JSONSerializer):
+    def to_internal_value(self, records):
+        if not all(isinstance(record, str) for record in records):
+            msg = 'Incorrect type. Expected a list of strings'
```

Comment:
The custom serializer here can be avoided, cf [this comment](https://github.com/desec-io/desec-stack/pull/50/files/e24690761137d31315045721ac1716b71fff7600#r123376800).